Directory.Packages.props

```diff
@@ -4,83 +4,83 @@
+    <PackageVersion Include="Azure.Messaging.ServiceBus" Version="7.12.0" />
     <PackageVersion Include="Azure.Storage.Blobs" Version="12.14.1" />
-    <PackageVersion Include="BenchmarkDotNet" Version="0.13.3" />
+    <PackageVersion Include="BenchmarkDotNet" Version="0.13.4" />
     <PackageVersion Include="Confluent.Kafka" Version="1.9.3" />
     <PackageVersion Include="Confluent.SchemaRegistry.Serdes.Avro" Version="1.9.3" />
     <PackageVersion Include="coverlet.collector" Version="3.1.2" />
     <PackageVersion Include="dapper.contrib" Version="2.0.78" />
     <PackageVersion Include="EntityFramework" Version="6.4.4" />
     <PackageVersion Include="Google.Protobuf" Version="3.21.12" />
-    <PackageVersion Include="Grpc.Core" Version="2.46.5" />
+    <PackageVersion Include="Grpc.Core" Version="2.46.6" />
     <PackageVersion Include="Grpc.Net.Client" Version="2.51.0" />
     <PackageVersion Include="Grpc.Tools" Version="2.51.0" />
-    <PackageVersion Include="Hangfire.AspNetCore" Version="1.7.32" />
-    <PackageVersion Include="Hangfire.Core" Version="1.7.32" />
+    <PackageVersion Include="Hangfire.AspNetCore" Version="1.7.33" />
+    <PackageVersion Include="Hangfire.Core" Version="1.7.33" />
     <PackageVersion Include="Hangfire.MemoryStorage" Version="1.7.0" />
     <PackageVersion Include="Iesi.Collections" Version="4.0.5" />
     <PackageVersion Include="Ionic.Zlib.Core" Version="1.0.0" />
     <PackageVersion Include="Marten" Version="5.11.0" />
     <PackageVersion Include="MathNet.Numerics" Version="5.0.0" />
     <PackageVersion Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
-    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.20.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
     <PackageVersion Include="Microsoft.AspNetCore.SignalR.Core" Version="1.1.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="6.0.8" />
-    <PackageVersion Include="Microsoft.Azure.Cosmos" Version="3.31.2" />
+    <PackageVersion Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="6.0.13" />
+    <PackageVersion Include="Microsoft.Azure.Cosmos" Version="3.32.0" />
     <PackageVersion Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
     <PackageVersion Include="Microsoft.Azure.WebJobs.Extensions.EventHubs" Version="5.1.2" />
     <PackageVersion Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="5.8.1" />
     <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" />
-    <PackageVersion Include="Microsoft.Data.SqlClient" Version="2.1.4" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.4.0" />
```

Comment:
I think this actually breaks an older version of Visual Studio, which is why it depends upon the older version. I don't know of any official guidance on this from Microsoft, but that's why it is on 3.8 instead of 4.x